newrelic_lambda_cli/integrations.py

```diff
@@ -322,26 +339,41 @@ def update_log_ingestion_function(input):
             if input.tags
             else [],
         )
-        client.get_waiter("stack_update_complete").wait(
-            StackName=nested_stack["PhysicalResourceId"]
-        )
 
-        click.echo("Removing outer stack")
+        try:
+            client.get_waiter("stack_update_complete").wait(
+                StackName=nested_stack["PhysicalResourceId"]
+            )
+        except botocore.exceptions.WaiterError as e:
+            failure(e.last_response["Status"]["StatusReason"])
```

Comment:
Should we also `raise e` here? Otherwise, we continue on, right?

Or does the call to failure terminate the process?

Reply:
It wasn't interrupting execution before, but it did occur to me as well that maybe it should. I can add a return statement here.

Reply:
Presumably it was throwing the WaiterError, which wasn't being caught, which not only terminated this function, but the caller and on up the stack, until it was caught.

Now, we catch the exception, and continue. That seems like a bug.